language/move-vm/runtime/src/runtime.rs

```diff
@@ -313,6 +337,18 @@ impl VMRuntime {
             log_context,
         )?;
 
+        let mut return_layouts = vec![];
+        for ty in return_tys.iter() {
+            let layout = self.loader.type_to_type_layout(ty).map_err(|_err| {
+                PartialVMError::new(StatusCode::INTERNAL_TYPE_ERROR)
+                    .with_message(
+                        "cannot be called with non-serializable return type(s)".to_string(),
```

Comment:
```suggestion
                        "cannot be called with non-serializable return type".to_string(),
```

since we'll only ever report one of these errors...